{
	"about": "https://jvndb.jvn.jp/ja/contents/2018/JVNDB-2018-016179.html",
	"title": "GitLab Community および Enterprise Edition における認証の欠如に関する脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2018/JVNDB-2018-016179.html",
	"description": "GitLab Community および Enterprise Edition には、認証の欠如に関する脆弱性が存在します。",
	"identifier": "JVNDB-2018-016179",
	"references": [
		{
			"text": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-20501",
			"source": "CVE",
			"id": "CVE-2018-20501"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2018-20501",
			"source": "NVD",
			"id": "CVE-2018-20501"
		},
		{
			"text": "https://cwe.mitre.org/data/definitions/862.html",
			"id": "CWE-862",
			"title": "認証の欠如(CWE-862)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:gitlab:gitlab",
			"version": "2.2",
			"vendor": "GitLab.org",
			"product": "GitLab"
		}
	],
	"cvss": [
		{
			"version": "2.0",
			"score": "6.5",
			"type": "Base",
			"severity": "Medium",
			"vector": "AV:N/AC:L/Au:S/C:P/I:P/A:P"
		},
		{
			"version": "3.0",
			"score": "6.3",
			"type": "Base",
			"severity": "Medium",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:L/A:L"
		}
	],
	"date": "2020-01-21T17:15+09:00",
	"issued": "2020-01-21T17:15+09:00",
	"modified": "2020-01-21T17:15+09:00"
}
